drivers/adc/adc_ads114s0x.c

```diff
@@ -597,9 +606,10 @@ static int ads114s0x_channel_setup(const struct device *dev,
 	uint8_t idac_magnitude = 0;
 	uint8_t idac_mux = 0;
 	uint8_t pin_selections[4];
+	uint8_t vbias = 0;
```

Comment:
These actually needs to be initialized because of how the access macro works right?

Reply:
yes